### PR TITLE
Changelog update for #713

### PR DESCRIPTION
**Pull request by usmansaleem**
Signed-off-by: Usman Saleem <usman@usmans.info>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,18 @@
 # Changelog
 
+## 1.4.5
+
+### Additions and Improvements
+
+ - Introduced Security Module Plugin API. This allows to switch to a different security module provider to
```
Comment (macfarla) [inline CHANGELOG.md:7]: This allows you to switch ...?
or
This allows use of a different ...?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,18 @@
 # Changelog
 
+## 1.4.5
+
+### Additions and Improvements
+
+ - Introduced Security Module Plugin API. This allows to switch to a different security module provider to 
+ provide cryptographic function that can be used by NodeKey (such as sign, ECDHKeyAgreement etc.). By default,
+  register KeyPairSecurityModule otherwise attempt to load Security Module via plugin API specified by
```
Comment (macfarla) [inline CHANGELOG.md:9]: are we saying KeyPairSecurityModule is loaded by default?

Comment (macfarla) [reply CHANGELOG.md:9]: also would be nice to be consistent with casing and terms ie "Security Module" here vs "security module provider" above.

**Comment by usmansaleem** [comment]
@macfarla changelog reworded, please have a look again.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,18 @@
 # Changelog
 
+## 1.4.5
+
+### Additions and Improvements
+
+ - Introduced SecurityModule plugin API. This allows use of a different security module as a plugin to 
+ provide cryptographic function that can be used by NodeKey (such as sign, ECDHKeyAgreement etc.). KeyPairSecurityModule
+  is registered and used by default. The CLI option `--security-module=<name>. (defaults to localfile)` can be used
```
Comment (macfarla) [inline CHANGELOG.md:9]:
```suggestion
  is registered and used by default. The CLI option `--security-module=<name> (defaults to localfile)` can be used 
```

Comment (usmansaleem) [reply CHANGELOG.md]: Thanks, removed the full stop!

**Pull request merged**